cde/sdk/go/Makefile

```diff
@@ -0,0 +1,133 @@
+# Make does not offer a recursive wildcard function, so here's one:
+rwildcard=$(wildcard $1$2) $(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2))
+
+SHELL := /bin/bash
+NAME := cde
+BUILD_TARGET = build
+MAIN_SRC_FILE=main.go
+GO := GO111MODULE=on go
+GO_NOMOD :=GO111MODULE=off go
+REV := $(shell git rev-parse --short HEAD 2> /dev/null || echo 'unknown')
+ORG := jenkins-x-plugins
+ORG_REPO := $(ORG)/$(NAME)
+RELEASE_ORG_REPO := $(ORG_REPO)
+ROOT_PACKAGE := github.com/$(ORG_REPO)
+GO_VERSION := 1.16
+GO_DEPENDENCIES := $(call rwildcard,pkg/,*.go) $(call rwildcard,cmd/,*.go)
+
+BRANCH     := $(shell git rev-parse --abbrev-ref HEAD 2> /dev/null  || echo 'unknown')
+BUILD_DATE := $(shell date +%Y%m%d-%H:%M:%S)
+CGO_ENABLED = 0
+
+REPORTS_DIR=$(BUILD_TARGET)/reports
+
+GOTEST := $(GO) test
+
+# set dev version unless VERSION is explicitly set via environment
+VERSION ?= $(shell echo "$$(git for-each-ref refs/tags/ --count=1 --sort=-version:refname --format='%(refname:short)' 2>/dev/null)-dev+$(REV)" | sed 's/^v//')
+
+# Build flags for setting build-specific configuration at build time - defaults to empty
+#BUILD_TIME_CONFIG_FLAGS ?= ""
+
+# Full build flags used when building binaries. Not used for test compilation/execution.
+BUILDFLAGS :=  -ldflags \
+  " -X $(ROOT_PACKAGE)/pkg/version.Version=$(VERSION)\
+		-X $(ROOT_PACKAGE)/pkg/version.Revision='$(REV)'\
+		-X $(ROOT_PACKAGE)/pkg/version.Branch='$(BRANCH)'\
+		-X $(ROOT_PACKAGE)/pkg/version.BuildDate='$(BUILD_DATE)'\
+		-X $(ROOT_PACKAGE)/pkg/version.GoVersion='$(GO_VERSION)'\
```

Comment:
We don't have a version package so all these flags do not really make sense. 
It would be nice to have it in future so I guess it's ok to keep it.

Reply:
@afrittoli I've removed this for now.. we can add it back when we need them.